draft-tuexen-opsawg-pcapng.xml

```diff
@@ -1371,6 +1400,37 @@ Section Header
 
             <t>Example: '0'.</t>
 
+            <t hangText="epb_packetid:"><vspace blankLines="0"/>The
+            epb_packetid option is a 64-bit unsigned integer that uniquely
+            identifies the packet. If the same packet is seen by multiple
+            interfaces and there is a way for the capture application to
+            correlated them, the same epb_packetid value must be used. An
+            example could be where an XDP program on ingress captures a packet,
+            which gets modified and then exits the XDP program. In this case,
+            two packets are in the capture file, which are not identical but
+            the epb_packetid can be used correlate them.</t>
+
+            <t>Example: '0'.</t>
+
+            <t hangText="epb_queue:"><vspace blankLines="0"/>The epb_queue
+            option is a 32-bit unsigned integer that identifies on which queue
+            of the interface the specific packet was received.</t>
+
+            <t>Example: '0'.</t>
+
+            <t hangText="epb_verdict:"><vspace blankLines="0"/>The epb_verdict
+            option stores a verdict of the packet. This verdict can be set by
+            various components, i.e. Hardware, Linux's eBPF TC or XPD
```

Comment:
Typo: XPD -> XDP

Reply:
> Eelco Chaudron <notifications@github.com> wrote: Not sure if I had to add my name as an editor if I added text. If not I can remove it, no problem. Generally, in an IETF document, we would acknowledge all contributors in the Acknolwedgements section, but only list the lead authors/editors in the heading. There is a guideline for five authors only on an RFC, and we'll justify or prune names if/when we get closer to that stage.
> […](#)

Ok, will leave it for now, or let me know if you want me to remove/move.

Reply:
Thanks! fixed in last commit